src/main.js

```diff
@@ -111,7 +138,7 @@ export default function connectToStores(...connectArgs) {
       handleStoresChange = () => {
         return this.setState({
           ...this.state,
-          customProps: connectArgs[0](this.takeSnapshot())
+          customProps: connectArgs[connectArgs.length - 1](this.takeSnapshot()),
```

Comment:
I should lint the project as-well, I'm not fan of trailing comas :+1:

If we need IE8-9 support we should also build the dist with es3ify-loader.